x-pack/qa/sql/src/main/resources/command.csv-spec

```diff
@@ -162,3 +162,27 @@ last_name            | VARCHAR
 last_name.keyword    | VARCHAR
 salary               | INTEGER
 ;
+
+
+describeIncludeExclude
```

Comment:
Since we use ES' own index names resolution, this means `_all` is supported as well. Can you introduce a test covering something like `SELECT * FROM _all, -.security*`?

Reply:
I'm hesitant to introduce `_all` since any meta index that might be introduced, will show-up later and the inclusion and exclusion is already tested here.